wordpress-mobile/WordPress-iOS
# My Site Dashboard: add the initial structure for the posts card

Part of #17697

This PR adds the initial structure to show 3 draft posts. It's not possible to interact with them neither the syncing is done, this will be tackled in future PRs.

I also want to decouple the `ViewController` and the `ViewModel`. I'll do it in a follow-up PR but please if you have any feedback go ahead!

<img src="https://user-images.githubusercontent.com/7040243/148850445-8db35bc9-637b-44fa-9069-81e99c5f697e.png" width="300">

### To test

1. Copy and paste [this gist](https://gist.github.com/leandroalonso/ba1c99305807e19839ba7b11a48c1a17) into `BlogViewController.m`
2. Run the app
3. Make sure that the first 3 draft posts appear
4. If they're not appearing, go to posts > draft
5. Close and reopen the app*

*Syncing is not available yet, so the posts need to be on the database.

## Regression Notes
1. Potential unintended areas of impact
n/a

2. What I did to test those areas of impact (or what existing automated tests I relied on)
n/a

3. What automated tests I added (or what prevented me from doing so)
I plan to add unit tests for the view model in an upcoming PR.

PR submission checklist:
